pkgs/tools/compression/lzip/default.nix

```diff
@@ -10,7 +10,7 @@ stdenv.mkDerivation rec {
   version = "1.23";
   outputs = [ "out" "man" "info" ];
 
-  nativeBuildInputs = [ texinfo ];
+  nativeBuildInputs = [ ];
 
```

Comment:
```suggestion
```
might as well

Reply:
Sounds reasonable! Proposed https://github.com/NixOS/nixpkgs/pull/211111 to clean it up.